### ISSUE TITLE
Questions:

### ISSUE THREAD

**Issue by SirTwsted** [Kyusung4698/PoE-Overlay#787]
Hi Kyusung, 
Why are you not using the official price store from GGG and only using Price Ninja and POE Prices Info  ??
Shouldn't you be using the official trading and store prices now from GGG at:
https://www.pathofexile.com/trade/search/Harvest

Im just curiously asking  :)

Also there seems to be some confusing.
My weapon says its 32% Increased Spell Damage -- But the overlay says its 28% Increased Spell Damage.
Also the weapon says 7.00% Critical Strike Chance but the overlay says 6~%
Attacks per Second 1.20 -- but your overlay says 1~%

Im confused, who is right POE/GGG or your Overlay ?? 

![Screenshot_24](https://user-images.githubusercontent.com/15247713/86297088-6e80a780-bbc8-11ea-92e4-4cc7eec93f3f.png)

I would also love a feature that tells me Exactly what DPS the weapon is actually doing Exactly.
even when I use Blacksmith's Whitestone on the weapon and any other crafting or augments done to the weapon.

Because if I compare the current weapon im using to one I found on the ground, 
I would like to know EXACTLY what DPS is the weapon,  to know if its worth using it or not.
There is a small app I use that does this, but im hoping you can implement this feature into your Overlay.

![Screenshot_26](https://user-images.githubusercontent.com/15247713/86297699-00d57b00-bbca-11ea-92ba-bbf05f0e0e1a.png)

Its from POE Item Info.
It basically tells me, 
1. - Max Sockets the item can have - So a player knows by looking if that item can do 3, 4, 5 or 6 sockets max.
2. - Total DPS from combined modifications and Prefixes.
3. - Prefix Total DPS
4. - Its original Base Level before the item was modified. and the3 level its currently at after the modifications.

Basically this is great when comparing items or weapons, to know exactly what DPS the weapon is actually doing.
And one can compare a weapon they found from drops, to make the decision if they want to upgrade their item or weapon with more DPS by this Info..

Here is another image example of a much higher weapon I found in a drop.
.
![Screenshot_27](https://user-images.githubusercontent.com/15247713/86298519-ef8d6e00-bbcb-11ea-983e-3889dac95aaf.png)

Thanks :)

**Comment by MalakaJones** [comment]
The reason why you dont have the exact values on your mods is that the affix range is probably set to 20% in the options. You want it to be 0. I suggest you to use the discord server for questions about the Overlay.
